### DISCUSSION TITLE
Any pvlib presentations at SciPy Conference 2022

### DISCUSSION THREAD

**Discussion by mikofski** [pvlib/pvlib-python#1383] [General]
Sorry for bombing irrelevant issues with this topic, I should have started this discussion here first.

[SciPy Conference 2022](https://www.scipy2022.scipy.org/) is happening in Austin from July 11 - 17. I think some topics that might be interesting would be:

1. initial storage support (small dev grant)
2. one stop shop for solar resource data (GSoC)

any other ideas?

**Comment by Peque** [comment]
@mikofski Do you know if traveling (being on-site) is required for that conference? (I mean for presenters of course)

**Reply by mikofski** [reply]
Seems like they're aiming for in-person, but perhaps that will change?
![image](https://user-images.githubusercontent.com/1385621/150383700-f36aa830-b46b-4e21-be80-e95340c8cf3d.png)

**Reply by mikofski** [reply]
Seems like it may still be in limbo:
![image](https://user-images.githubusercontent.com/1385621/150384089-1eb24038-b64e-4f1b-8529-99b4201e16e8.png)

**Comment by kandersolar** [comment]
An idea rolling around the back of my head:  survey industry folks on how they use the various open-source PV packages internally.  I very recently had my mind boggled by learning that pvlib is deployed in the field for real-time calculations in solar farm controllers.  I'd personally love to hear more stories like that, but the broader goal would be showcasing that open source PV software actually gets used in the real world, is supporting the clean energy transition, and is worth continued development and maintenance effort.  Might be hard to get companies to spill secrets, but I expect we could get some high-level info from some of them at least. 

Not sure scipy is the best venue for something like that though, need to think about it.  Plus I'm not excited about in-person anyway at the moment.

**Reply by cwhanse** [reply]
Somewhere, on a desk I don't use now, is a list of companies or commercial products that use pvlib-python, based on conversations at meetings. Most people I asked were open about using pvlib but didn't mention it in their product literature (Clean Power Research is a notable exception). I think it just doesn't occur to developers or marketers to acknowledge the open-source components, in a similar way that many of us don't acknowledge using e.g. matplotlib to make figures for a paper. I took this as a sign of pvlib's success and maturity.

**Reply by mikofski** [reply]
I have been rolling the survey idea around in my head for a while, but TBH, I think design and process of survey is not trivial - and was wondering if this could be a use for a small dev grant or just ask for help from NF community. But I suppose a survey topic should be started in a separate thread.

**Comment by campanelli-sunpower** [comment]
SunPower uses pvlib-python in production. We probably could provide a brief sketch of how we incorporate it.

**Reply by cwhanse** [reply]
Any chance you could talk your leadership into using one of the logos (Citing section of the homepage here) somewhere?

**Reply by campanelli-sunpower** [reply]
That would be nice. Checking :)

**Reply by campanelli-sunpower** [reply]
I got the ok to run this by SunPower's comm's group. Can you be more specific about which page(s) the SunPower logo would appear on? I think they might want to see this in advance.

**Reply by cwhanse** [reply]
Hi Mark, I was suggesting the opposite - that perhaps SunPower could display the pvlib logo, as a way to acknowledge pvlib's use. 

But that's an interesting idea - could pvlib host a page displaying logos of companies or products that use pvlib? We'd have to think if that would imply some kind of endorsement (avoidable with a few words). An adopter's page would serve pvlib's interest (better knowledge of the user community), and if it also acts as advertising for adopters, I don't see a problem with that.

**Reply by kandersolar** [reply]
> could pvlib host a page displaying logos of companies or products that use pvlib? 

I like this idea too.  I've seen some other scientific packages do that.  https://fluxml.ai/ is one example.  I assume we'd need to get permission from each entity to use their logo, right?

**Reply by campanelli-sunpower** [reply]
@cwhanse I'm not sure what level of visibility we could achieve at SunPower. I'm happy to put the pvlib and pvfactors logo in the PVAPI+SimEng2 documentation to make their usage more prominently displayed. That would essentially only be seen by users of the tool. Finding an appropriate place somewhere outside of the documentation seems like it would be a "contextual stretch".

I do think you'll get much more visibility putting somewhere on a pvlib-python page. I promoted that to management because I figure it's good for the project to have DoE to see their funding at work in a US company, as well as provide additional credibility for potential pvlib users. Let me know if you come up with a proposed location, and I will be happy to forward it along.

**Comment by mikofski** [comment]
SciPy remote attendee access announcement coming soon:  https://twitter.com/SciPyConf/status/1494031996070371332
